### PR TITLE
chore: remove deprecated testnets

### PR DESCRIPTION
**Pull request by lynnshaoyu**
This PR removes the following permanently deprecated testnets: 
ROPSTEN = 3
RINKEBY = 4
KOVAN = 42
OPTIMISTIC_KOVAN = 69
ARBITRUM_RINKEBY = 421611

### CODE REVIEW

**Review by zhongeric — APPROVED** [review]
lgtm! note that we'll have to make similar changes in routing-api

**Comment by lynnshaoyu** [comment]
> lgtm! note that we'll have to make similar changes in routing-api

ty! already have a draft PR :D https://github.com/Uniswap/routing-api/pull/238

File: src/providers/v2/subgraph-provider.ts [hunk 1]
```diff
@@ -39,8 +39,6 @@ type RawV2SubgraphPool = {
 const SUBGRAPH_URL_BY_CHAIN: { [chainId in ChainId]?: string } = {
   [ChainId.MAINNET]:
     'https://api.thegraph.com/subgraphs/name/ianlapham/uniswapv2',
-  [ChainId.RINKEBY]:
-    'https://api.thegraph.com/subgraphs/name/ianlapham/uniswap-v2-rinkeby',
```
Comment (jsy1218) [inline src/providers/v2/subgraph-provider.ts:43]: wow didn't realize we have a provider url based on personal employee's account..

Comment (jsy1218) [reply src/providers/v2/subgraph-provider.ts:43]: no action item for now I think.

**Pull request merged**